RecoEgamma/ElectronIdentification/plugins/cuts/GsfEleHadronicOverEMEnergyScaledCut.cc

```diff
@@ -0,0 +1,64 @@
+#include "PhysicsTools/SelectorUtils/interface/CutApplicatorWithEventContentBase.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectron.h"
+
+class GsfEleHadronicOverEMEnergyScaledCut : public CutApplicatorWithEventContentBase {
+public:
+  GsfEleHadronicOverEMEnergyScaledCut(const edm::ParameterSet& c) :
+    CutApplicatorWithEventContentBase(c),
+    _barrelC0(c.getParameter<double>("barrelC0")),
+    _barrelCE(c.getParameter<double>("barrelCE")),
+    _barrelCr(c.getParameter<double>("barrelCr")),
+    _endcapC0(c.getParameter<double>("endcapC0")),
+    _endcapCE(c.getParameter<double>("endcapCE")),
+    _endcapCr(c.getParameter<double>("endcapCr")),
+    _barrelCutOff(c.getParameter<double>("barrelCutOff")) 
+  {
+    edm::InputTag rhoTag = c.getParameter<edm::InputTag>("rho");    
+    contentTags_.emplace("rho",rhoTag);  
+    
+  }
+  
+  result_type operator()(const reco::GsfElectronPtr&) const final;
+
+  void setConsumes(edm::ConsumesCollector&) final;
+  void getEventContent(const edm::EventBase&) final;
+
+  double value(const reco::CandidatePtr& cand) const final;
+
+  CandidateType candidateType() const final { 
+    return ELECTRON; 
+  }
+
+private:
+  const float _barrelC0, _barrelCE, _barrelCr, _endcapC0, _endcapCE, _endcapCr, _barrelCutOff;  
```

Comment:
leading underscore should be avoided in variables names (CMS style). Trailing underscore is OK.

Reply:
Changes implemented as required: https://github.com/lsoffi/cmssw/blob/CMSSW_9_4_0_pre3_TnP/RecoEgamma/ElectronIdentification/plugins/cuts/GsfEleHadronicOverEMEnergyScaledCut.cc